front/src/applications/operationalStudies/views/ManageTrainSchedule.tsx

```diff
@@ -52,40 +50,84 @@ export default function ManageTrainSchedule({ setDisplayTrainScheduleManagement
       }),
     }
   );
+  const tabRollingStock = {
+    title: (
+      <>
+        <img src={rollingStockPic} alt="rolling stock" height={24} />
+        <span className="ml-2">{t('tabs.rollingStock')}</span>
+      </>
+    ),
+    withWarning: rollingStockID === undefined,
+    label: t('tabs.rollingStock'),
+    content: <RollingStockSelector />,
+  };
+
+  const tabPathFinding = {
+    title: (
+      <>
+        <img src={pahtFindingPic} alt="path finding" height={24} />
+        <span className="ml-2">{t('tabs.pathFinding')}</span>
+      </>
+    ),
+    withWarning: pathFindingID === undefined,
+    label: t('tabs.pathFinding'),
+    content: (
+      <div className="row no-gutters">
+        <div className="col-xl-6 pr-xl-2">
+          {mustUpdatePathfinding !== undefined && <Itinerary mustUpdate={mustUpdatePathfinding} />}
+        </div>
+        <div className="col-xl-6">
+          <div className="osrd-config-item mb-2">
+            <div className="osrd-config-item-container osrd-config-item-container-map">
+              <Map />
+            </div>
+          </div>
+        </div>
+      </div>
+    ),
+  };
+
+  const tabSimulationSettings = {
+    title: (
+      <>
+        <img src={simulationSettings} alt="simulation settings" height={24} />
+        <span className="ml-2">{t('tabs.simulationSettings')}</span>
+      </>
+    ),
+    label: t('tabs.simulationSettings'),
+    content: (
+      <div className="row no-gutters">
+        <div className="col-lg-6 pr-lg-2">
```

Comment:
(Suggestion)
Remove one of the 2 divs

Reply:
Bootstrap css conception impose use of 2 divs to respect css implementation.